gateway/1.10/modules/ROOT/pages/_partials/flex-otel-support.adoc

```diff
@@ -0,0 +1,18 @@
+== Distributed Tracing Overview
+
+OpenTelemetry provides a unified standard for instrumenting, generating, collecting, and exporting telemetry data across distributed systems. Flex Gateway supports OpenTelemetry, enabling you to configure the generation of distributed tracing data for all API traffic managed by the gateway. This integration delivers deep visibility into API request flows, making it easier to analyze system behavior, troubleshoot issues, and understand interactions between Flex Gateway APIs and external services.
+
+With the Anypoint Monitoring Telemetry Exporter, you can send Flex Gateway trace data to third-party observability and analytics platforms, including Azure Monitor, Splunk HEC, or any OpenTelemetry-compliant tool. This capability allows you to leverage end-to-end observability, correlating API traces with data from other systems to build a comprehensive view of your environment.
+
+== Before You Begin
+
+To use the OpenTelemetry with Flex Gateway, you must be familiar with:
+
+* Distributed tracing concepts, including traces and spans. 
++
+If you're new to distributed tracing and OpenTelemetry, we recommend starting with https://opentelemetry.io/docs/concepts/what-is-opentelemetry/[What is OpenTelemetry] and https://opentelemetry.io/docs/concepts/observability-primer/#understanding-distributed-tracing[Understanding distributed tracing].
+* xref:monitoring::telemetry-exporter.adoc[How to export telemetry data to third-party monitoring systems].
```

Comment:
Flagging because I don't think the page title is needed, but you might test it to make sure.